ocs_sample_library_preview/SDS/SdsInterpolationMode.py

```diff
@@ -1,12 +1,13 @@
 from enum import Enum
 
 
-class SdsStreamMode(Enum):
+class SdsInterpolationMode(Enum):
     """
-    Enum 0-3
+    enum 0-5 fully inclusive
     """
     Continuous = 0
     StepwiseContinuousLeading = 1
     StepwiseContinuousTrailing = 2
     Discrete = 3
-    Default = Continuous
+    ContinuousNullableLeading = 4
+    ContinuousNullableTrailing = 5
```

Comment:
Do we want the default still?

Reply:
I'm not sure that it matters, but I don't even remember removing this in the first place so I'll add it back.